examples/java/pom.xml

```diff
@@ -267,6 +267,11 @@
                     <include>WordCountIT.java</include>
                     <include>WindowedWordCountIT.java</include>
                   </includes>
+                  <classpathDependencyExcludes>
```

Comment:
Is there a JIRA you can link here about separating the precommit runs and eliminating the `jenkins-precommit` profile?

Reply:
https://issues.apache.org/jira/browse/BEAM-2596 ?